packages/react-query-observable/src/queryFn/observableQueryFn.ts

```
@@ -1,14 +1,13 @@
 import { hashKey, type QueryCacheNotifyEvent, type QueryFunction, type QueryKey } from '@tanstack/react-query'
-import { type Observable } from 'rxjs'
+import { finalize, type Observable } from 'rxjs'
```

ESLint

[error] Unable to resolve path to module 'rxjs'.

(import-x/no-unresolved)